code/WorkInProgress/MechanicMadness.dm

```diff
@@ -344,6 +344,7 @@
 	w_class = 1.0
 	level = 2
 	var/cabinet_banned = false // whether or not this component is prevented from being anchored in cabinets
+	var/one_per_tile = false // if true makes it so that only one component can be wrenched on the tile
```

Comment:
```suggestion
	/// whether or not this component is prevented from being anchored in cabinets
	var/cabinet_banned = FALSE
	/// if true makes it so that only one component can be wrenched on the tile
	var/one_per_tile = FALSE
```